lib/alephant/broker/load_strategy/s3/base.rb

```diff
@@ -33,8 +33,8 @@ def load(component_meta)
 
           protected
 
-          def headers(component_meta)
-            Hash.new
```

Comment:
Really strange `Hash.new` was used there. Be worth ensuring that there wasn't some special behaviour that you get from a `Hash` object that you don't from `{}`?